disk/datadog_checks/disk/disk.py

```diff
@@ -78,10 +78,13 @@ def check(self, instance):
 
             # Get disk metrics here to be able to exclude on total usage
             try:
-                disk_usage = timeout(5)(psutil.disk_usage)(part.mountpoint)
+                disk_usage = timeout(self._timeout)(psutil.disk_usage)(part.mountpoint)
```

Comment:
(not a blocker) 

Would be nice to have a test for this configuration (1. check timeout method is called with timeout config. 2. check  warning is logged and check continues in case of TimeoutException)

Possibly by mocking the `timeout` and `psutil.disk_usage` methods.

Reply:
Done, re-requested your review to check that the tests make sense.